themes/default/layouts/page/get-started.html

```diff
@@ -0,0 +1,82 @@
+{{ define "hero" }}
+    {{ partial "hero.html" (dict "title" .Params.title) }}
+{{ end }}
+
+{{ define "main" }}
+    <section class="container mx-auto mt-20">
```

Comment:
```suggestion
    <section class="container relative mx-auto mt-20">
```

☝️ That will fix the shape positioning. The shape backgrounds are absolute positioned so the parent container of a shape background will need to have `position: relative` for the shapes to align correctly. Without the `relative` class the `top, right, bottom, left` positions will be based off the entire page instead of the parent container.